mtp_noms_ops/templates/security/policy-change-info.html

```diff
@@ -54,6 +44,10 @@ <h2 class="heading-medium">{% trans 'What you need to do' %}</h3>
         <li>{% trans 'transferring your money from prison to a new or existing bank account' %}</li>
       </ul>
 
+      <h2 class="heading-large">{% trans 'Prisoner bank accounts' %}</h2>
+
+      <p>{% trans 'Credits will not go into a prisoner's account if it holds £900 or more.' %}</p>
```

Comment:
you need `’` instead of `'` in the middle here… but you probs already saw circleci complain!